protobuf/schema/v1/schema.proto

```diff
@@ -8,7 +8,39 @@ option csharp_namespace = "OpenFeature.Flagd.Grpc";
 option php_namespace = "OpenFeature\\Providers\\Flagd\\Schema\\Grpc";
 
 import "google/protobuf/struct.proto";
-import "google/protobuf/empty.proto";
+
+// Request body for bulk flag evaluation, used by the ResolveAll rpc.
+message ResolveAllRequest {
+
+    // Object structure describing the EvaluationContext used in the flag evaluation, see https://docs.openfeature.dev/docs/reference/concepts/evaluation-context
+    google.protobuf.Struct context = 1;
+}
+
+// Response body for bulk flag evaluation, used by the ResolveAll rpc.
+message ResolveAllResponse {
+
+    // Object structure describing the evaluated flags for the provided context.
+    map<string, AnyFlag> flags = 1;
+}
+
+// A variant type flag response.
+message AnyFlag {
+
+    // The reason for the given return value, see https://docs.openfeature.dev/docs/specification/types#resolution-details
+    string reason = 1;
+
+    // The variant name of the returned flag value.
+    string variant = 2;
+
+    // The response value of the boolean flag evaluation, will be unset in the case of error.
+    oneof value {
+        bool                    bool_value = 3;
+        string                  string_value = 4;
+        int64                   int_value = 5;
```

Comment:
I'm in favour of dropping the `int` type from this definition, all values are stored in the evaluator as `float64`, as a result we don't actually know the 'destination' type in the case of bulk evaluation.

Reply:
That makes sense.

Reply:
resolved.